src/consumer/kafka.py

```
@@ -0,0 +1,332 @@
+                                await self._send_to_dlq(topic, msg.value, str(e))
+                                await self.consumer.commit({tp: msg.offset + 1})
+                                """
+                                
+                                """
+                            finally:
+                                """
+                                
+                                """
+                                if not success:
+                                    await asyncio.sleep(random.uniform(1, 3)) # avoid hammering both the broker and our logs.
+                                    break
```

Ruff (0.8.2)

`break` inside `finally` blocks cause exceptions to be silenced

(B012)